python/cudf/cudf/tests/test_gpu_arrow_parser.py

```diff
@@ -178,6 +179,7 @@ def make_gpu_parse_arrow_cats_batch():
 
 
 def test_gpu_parse_arrow_cats():
+    pytest.xfail(reason="need dictionary mapping in libcudf from_arrow")
```

Comment:
We need to remove this xfail before merging

Reply:
Yup